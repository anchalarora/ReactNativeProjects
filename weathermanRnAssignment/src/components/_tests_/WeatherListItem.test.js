import { shallow } from 'enzyme'
import React from 'react'
import renderer from 'react-test-renderer'

import WeatherListItem from '../weatherListItem'


it('should load flat list item view without crashing', () => {
    const rendered = renderer.create(<WeatherListItem />).toJSON()
    expect(rendered).toBeTruthy()
})


it('should render the Text element', () => {
    const component = shallow(<WeatherListItem item1='Humidity: 69' item2='Temp: 297.53' />)
    expect(component).toMatchSnapshot()
})
