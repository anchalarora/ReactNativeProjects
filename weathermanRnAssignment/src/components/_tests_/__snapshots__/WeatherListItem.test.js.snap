// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render the Text element 1`] = `ShallowWrapper {}`;
